docs/site/migration/models/mixins.md

```diff
@@ -552,10 +552,12 @@ export interface FindByTitleControllerMixinOptions {
  */
 export function FindByTitleControllerMixin<
   M extends Model,
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  T extends Constructor<any> = Constructor<object>
+  T extends MixinTarget<object>
```

Comment:
Should we use `object` consistently for all `MixinTarget<Object>`?

Reply:
Good catch, I agree to use `MixinTarget<object>` everywhere :+1: